Cargo.toml

```diff
@@ -15,8 +15,8 @@ path = "src/passthrough.rs"
 tokio = { version = "1.17.0", features = ["full"] }
 futures-util = { version = "0.3.21" }
 
-zeroconf = { git = "https://github.com/zeyugao/zeroconf-rs", optional = true }
-mdns = "3.0.0"
+zeroconf = { version = "0.14.2", optional = true }
```

Comment:
It looks like the newest version of zeroconf is 0.15. Is there a reason we're not targeting the latest version?

Reply:
Not at all.